### ISSUE TITLE
Test server issue

### ISSUE THREAD

**Issue by corneliusroemer** [yatisht/usher#282]
I just uploaded some 40 sequences and got this result. Can't open the link, somewhat strange. Thought I let you know:

<img width="1800" alt="image" src="https://user-images.githubusercontent.com/25161793/195992319-c9b369c6-2789-4f83-b6ae-23440c9e508a.png">

No console errors.

**Comment by AngieHinrichs** [comment]
Yes, there was a server-side SEGV caused by a strange little corner case in my code that generates a VCF file with usher's imputed alleles for ambiguous bases in the uploaded FASTA (for a custom track in the Genome Browser).  Thanks for letting me know!  Should be fixed now on the test server.
